keps/sig-cli/kustomize-secret-generator-plugins.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Kustomize Secret Generator Plugins
+authors:
+  - "@sethpollack"
+owning-sig: sig-cli
+reviewers:
+  - "@monopole"
+  - "@Liujingfang1"
+approvers:
+  - "@monopole"
+  - "@Liujingfang1"
```

Comment:
Should the approvers be the kustomize admin and/or maintainers? https://github.com/kubernetes-sigs/kustomize/blob/master/OWNERS_ALIASES

Reply:
to answer a question from @lavalamp, it is my intention that `apporvers` and `reviewers` are the people who _did_ not the people who _can_ which is the problem that the `OWNERS` mechanism addresses

Reply:
@calebamiles Then the fields should be called "reviewed-by" and "approved-by"? And it is weird to have those people added by the author rather than add themselves. I would not be surprised if we have many KEPs created with a different understanding of those fields.